tests/04_api/test_project_apply_api.py

```diff
@@ -631,3 +633,65 @@ async def test_project_apply_slurm_account(
             json=dict(slurm_account="NOT IN THE LIST"),
         )
         assert res.status_code == 422
+
+
+async def test_rate_limit(
```

Comment:
I'd suggest that only for this test we set `FRACTAL_API_SUBMIT_MIN_WAIT=1`, and we set it to 0 by default in all other tests (through the usual fixture). In this way the other tests remain unchanged, and we don't need to add any `sleep`.